README.md

```diff
@@ -265,31 +265,36 @@ compdef kubecolor=kubectl
 
 #### fish
 
-Fish completion is officially unsupported by `kubectl`, so it is unsupported by `kubecolor` as well.
+Fish completion is officially supported by `kubectl`.
+If you did not install `kubectl` with `homebrew` (`homebrew` will auto enable autocompletion), make sure the following line is in your `~/.config/fish/config.fish` file:
 
-However, there are 2 ways we can make them work. Please keep in mind these are a kind of "hack" and not officially supported.
-
-1. Use [evanlucas/fish-kubectl-completions](https://github.com/evanlucas/fish-kubectl-completions) with `kubecolor`:
-
-   * Install `kubectl` completions (https://github.com/evanlucas/fish-kubectl-completions):
+```fish
+kubectl completion fish | source
+```
 
-      ```fish
-      fisher install evanlucas/fish-kubectl-completions
-      ```
+To enable autocompletion for kubecolor, add the following function to your `~/.config/fish/config.fish` file:
 
-   * Add the following function to your `~/.config/fish/config.fish` file:
+```fish
+function kubecolor --wraps kubectl
+  command kubecolor $argv
+end
+```
 
-      ```fish
-      function kubectl
-        kubecolor $argv
-      end
-      ```
+If you want to substitute your `kubectl` with `kubecolor`, then also add the following function to your `~/.config/fish/config.fish` file:
 
-2. Use [awinecki/fish-kubecolor-completions](https://github.com/awinecki/fish-kubecolor-completions)
+```fish
+function kubectl --wraps kubectl
+  command kubecolor $argv
+end
+```
 
-   The first way will override `kubectl` command. If you wish to preserve both `kubectl` and `kubecolor` with completions, you need to copy [evanlucas/fish-kubectl-completions](https://github.com/evanlucas/fish-kubectl-completions) for the `kubecolor` command.
+If you want to use an alias like `k` for `kubecolor`, then also add the following function to your `~/.config/fish/config.fish` file:
 
-   For this purpose, you can use [awinecki/fish-kubecolor-completions](https://github.com/awinecki/fish-kubecolor-completions).
+```fish
+function k --wraps kubectl
+  command kubecolor $argv
+end
+```
```

Comment:
nitpick: This is very explicit, but due to the repetition it gets a bit difficult to read.

What about merging these all into the same code block?

(I can't add a review code suggestion because GitHub says "Applying suggestions on deleted lines is not supported.", so here's just a normal suggestion instead)

````markdown
To enable completions for `kubecolor`, add the following function to your `~/.config/fish/config.fish` file:

```fish
# reuse "kubectl" completions on "kubecolor"
function kubecolor --wraps kubectl
  command kubecolor $argv
end

# adds alias for "kubectl" to "kubecolor" with completions
function kubectl --wraps kubectl
  command kubecolor $argv
end

# adds alias for "k" to "kubecolor" with completions
function k --wraps kubectl
  command kubecolor $argv
end
```
````